src/content/docs/en/pages/architectures/api-gateways/api-gateways-security.mdx

```diff
@@ -0,0 +1,80 @@
+permalink: >-
+  /documentation/architecture/api-gateways/implement-api-gateways-security/
+meta_tags: >-
+  API Gateway Security, security perimeter, API Gateways, Edge Firewall, DDoS protection, WAF rules, security policies, Azion Edge Platform, traffic encryption, malicious activity
+namespace: docs_arch_api_gateways_security
+---
+
+API Gateways enable intelligent management and routing of API requests to the appropriate backend microservices across distributed networks. They act as a single entry point for all API requests. While an API Gateway facilitates efficient traffic flow and centralizes control, it also introduces the need for security measures and monitoring to prevent performance issues and security vulnerabilities.
+
+Defining a robust security perimeter ensures that only validated requests pass through, enforcing access control, traffic encryption, and threat detection. With this extra layer of protection, you're able to prevent unauthorized access, mitigate attacks, and secure sensitive data. This way, your gateway manages traffic and turns into a checkpoint to avoid malicious activity reaching your origins.
+
+Through this perimeter, you can implement solutions such as:
+
+- Rate limiting and throttling to control the volume of API requests.
+- Customized WAF rules to address the specific needs of your infrastructure.
+- Detection of DDoS attacks, bad bots, and other common threats.
+- Authentication mechanisms to secure your edge applications.
+- Observability tools to monitor performance and identify threats.
+
+---
+
+## API Gateway Security Architecture Diagram
+
+![API Gateway Security Architecture Diagram](/assets/docs/images/uploads/api-gateways-security-flow.png)
+
+### API Gateway Security Dataflow
+
+1. A request is sent to the API Gateway.
+    - In the case of the user, it can be a content request.
+    - The customer can be a developer using Azion's platform as an API security perimeter trying to configure and manage policies.
+2. The API Gateway processes the request and redirects it to the Azion Edge Platform.
+3. Edge Firewall verifies and analyzes the request to make sure is safe, including Rules Engine Rules, Edge Functions, WAF Rules, DDoS mitigation, network list policies, and more.
+    - In this step, any authentication account management policy is also applied, such as [Multi-Factor Authentication (MFA)](/en/documentation/products/accounts/multi-factor-authentication/), [account lockout policy](/en/documentation/products/accounts/account-lockout-policy/), and more. 
+4. Marketplace Integrations for security, such as Bot Manager or Rate Limit, are executed to complete the analysis and verification.
+    - Based on the firewall, account management, and integrations' configurations, the system denied any unauthorized request. Otherwise, the request continues the flow.
```

Comment:
```suggestion
    - Based on the firewall, account management, and integrations' configurations, the system denies any unauthorized request. Otherwise, the request continues the flow.
```